src/datepicker-input/datepicker-input.component.ts

```diff
@@ -84,4 +88,13 @@ export class DatePickerInput {
 	@Input() invalidText: string;
 
 	@Input() skeleton = false;
+
+	onChange() {
+		this.valueChange.emit(this.dateInput.nativeElement.value);
```

Comment:
Since we're binding directly to the element, we could just pass down the event and pull the value from `event.target` ... then we can also remove the extra `dateInput` bindings 👍